iOSClient/More/NCMore.swift

```diff
@@ -262,7 +262,7 @@ class NCMore: UIViewController, UITableViewDelegate, UITableViewDataSource {
         if indexPath.section == 0 {
             return 100
         } else {
-            return NCGlobal.shared.heightCellSettings
+            return UITableView.automaticDimension
```

Comment:
This alone a) doesn't support multilines, and b) reduces the current cell height.
To make this work you would need to
1. Add: `tableView.estimatedRowHeight = NCGlobal.shared.heightCellSettings` to the `viewDidLoad`
2. (?) To keep the old height, add Top & Bottom constraints to the label, to (somewhat) keep a min. height of 50pt.